dags/openshift_nightlies/util/manifest.py

```diff
@@ -128,7 +129,7 @@ def _get_dependencies(self):
 
     def _get_schedule_for_platform(self, platform):
         schedules = self.yaml['dagConfig']['schedules']
-        if bool(schedules.get("enabled", False) and var_loader.get_git_user() == "cloud-bulldozer"):
+        if bool(schedules.get("enabled", False)): # and var_loader.get_git_user() == "cloud-bulldozer"):
```

Comment:
this needs to be re-added so that playgrounds don't run on a schedule by default

Reply:
yes, will enable it before merging